src/views/about.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+    <meta name="description" content="This is the about page for Aggie Coding Club.">
+    <meta charset="UTF-8">
+</head>
+
```

Comment:
Consider adding the <body> tag around the <ul> element. Other than that it looks good to me! 👍

Reply:
LOL I totally forgot about that. Thanks, I added it!